tools/cldr-apps/js/src/esm/cldrVueMap.mjs

```diff
@@ -24,7 +26,7 @@ const specialToComponentMap = {
   add_user: AddUser,
   auto_import: AutoImport,
   downgraded: DowngradedVotes,
-  general: GeneralInfo,
+  general: GeneralInfo, // see cldrLoad.GENERAL_SPECIAL
```

Comment:
we could integrate this better, but this is sufficient for now.

Reply:
Really all of these keys could benefit from encapsulation. I'm not sure what how/where, though... using `cldrLoad.GENERAL_SPECIAL` in place of `general` for the key isn't allowed in the initializer, but we could have initialization code like
```
specialToComponentMap[cldrLoad.GENERAL_SPECIAL] = GeneralInfo;
```

and likewise for the other keys...